Tasks/JenkinsDownloadArtifacts/jenkinsdownloadartifacts.ts

```diff
@@ -27,24 +28,24 @@ class Credential {
 function getRequest(url: string, cred: Credential, strictSSL: boolean): Q.Promise<any> {
     const defer = Q.defer<any>();
 
-    request.get({url: url, strictSSL: strictSSL}, (err, res, body) => {
-                if (res && body && res.statusCode === 200)  {
-                    defer.resolve(JSON.parse(body));
-                } else {
-                    if (res && res.statusCode) {
-                        tl.debug(tl.loc('ServerCallErrorCode', res.statusCode));
-                    }
-                    if (body) {
-                        tl.debug(body);
-                    }
-                    defer.reject(new Error(tl.loc('ServerCallFailed')));
-                }
-            })
-            .auth(cred.mUsername, cred.mPassword, true)
-            .on('error', (err) => {
-                //TODO: Do we even need an 'error' handler here if we're just re-throwing?
-                defer.reject(new Error(err.message));
-            });
+    request.get({ url: url, strictSSL: strictSSL }, (err, res, body) => {
+        if (res && body && res.statusCode === 200) {
+            defer.resolve(JSON.parse(body));
+        } else {
+            if (res && res.statusCode) {
+                tl.debug(tl.loc('ServerCallErrorCode', res.statusCode));
```

Comment:
Don't localize debug messages. You don't want to investigate with a log in a language you don't understand :)